lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -322,6 +388,92 @@ public void OnError(Exception error)
             }
         }
 
+        private sealed class BreakTestTask : ITask, IObserver<ICloseEvent>
```

Comment:
I updated comments and task name for this task. For other task, I added comments. Rename would cause more changes everywhere that would disturb people from reviewing this PR.